segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java

```diff
@@ -55,6 +66,9 @@
     private final String dbDir;
     private final String logId;
     private final Consumer<String> closeCallback;
+    private final Integer writeBufferSizeMB;
```

Comment:
"int", not "Integer".

fix below too

Reply:
Changed in `RocksDBCache.java` and `RocksDBConfig.java`.